docs/001_develop/03_client-capabilities/006_charts/002_g2plot-chart/01_chart-types.mdx

```diff
@@ -0,0 +1,938 @@
+
+Available [Chart types](./docs/api/g2plot-chart.charttypes.md) include:
+
+- **Line** - for continuous data trends
+- **Area** - ideal for visualizing cumulative values
+- **Column** and **Bar** - for category comparisons
+- **Pie** and **Donut** - representing parts of a whole
+- **Dualaxes** - for layered, multi-metric displays
+- **Rose** - useful for categorical distribution
+- **Scatter** - showing relationships between variables
+- **Stock** - tracking stock performance over time
+- **Mix** - combining multiple chart types for complex data
+
+By choosing an appropriate chart type, you can effectively highlight patterns, comparisons, or distributions, enabling viewers to gain clear insights from the data.
+
+## Examples
+
+### Line
+
+Shows trends over time or continuous data, ideal for tracking changes.
+
+<Tabs defaultValue="declaration" values={[{ label: 'Declaration', value: 'declaration', }, { label: 'Configuration', value: 'config', }, { label: 'Sample data', value: 'data', }]}>
+
+<TabItem value="declaration">
+```html
+<rapid-g2plot-chart 
+  type="line"
+  :config=${chartConfiguration}
+  :data=${chartData}
+></rapid-g2plot-chart>
+```
+
+</TabItem>
+<TabItem value="config">
+```typescript
+const chartConfiguration = {
+  width: 700,
+  padding: "auto",
+  xField: "groupBy",
+  yField: "value",
+  xAxis: {
+    tickCount: 5,
+  },
+};
+```
+</TabItem>
+<TabItem value="data">
+```typescript
+const chartData = [
+  {
+    groupBy: "2010-01",
+    value: 1998,
+  },
+  {
+    groupBy: "2010-02",
+    value: 1850,
+  },
+  {
+    groupBy: "2010-03",
+    value: 1720,
+  },
+  {
+    groupBy: "2010-04",
+    value: 1818,
+  },
+  {
+    groupBy: "2010-05",
+    value: 1920,
+  },
+  {
+    groupBy: "2010-06",
+    value: 1802,
+  },
+  {
+    groupBy: "2010-07",
+    value: 1945,
+  },
+  {
+    groupBy: "2010-08",
+    value: 1856,
+  },
+  {
+    type: "2010-09",
+    value: 2107,
+  },
+  {
+    groupBy: "2010-10",
+    value: 2140,
+  },
+  {
+    groupBy: "2010-11",
+    value: 2311,
+  },
+  {
+    groupBy: "2010-12",
+    value: 1972,
+  },
+  {
+    groupBy: "2011-01",
+    value: 1760,
+  },
+  {
+    groupBy: "2011-02",
+    value: 1824,
+  },
+  {
+    groupBy: "2011-03",
+    value: 1801,
+  },
+];
+```
+</TabItem>
+</Tabs>
+
+<LineChartExample/>
+
+### Area
+
+Highlights cumulative data over time, useful for displaying totals or progress.
+
+<Tabs defaultValue="declaration" values={[{ label: 'Declaration', value: 'declaration', }, { label: 'Configuration', value: 'config', }, { label: 'Sample data', value: 'data', }]}>
+
+<TabItem value="declaration">
+```html
+<rapid-g2plot-chart 
+  type="area"
+  :config=${chartConfiguration}
+  :data=${chartData}
+></rapid-g2plot-chart>
+```
```

Comment:
Can I just double check with you that the bindings don't need to be functions, I thought they usually do in FAST? Same for other ones in this file
```suggestion
<rapid-g2plot-chart 
  type="area"
  :config=${() => chartConfiguration}
  :data=${() => chartData}
></rapid-g2plot-chart>
```
```